linters commit

uzairali19 opened the pull request.

eypsrcnuygr reviewed (changes requested):
## Status: Changes Required :recycle: 

Hello @uzairali19. I've checked your page and it looks good. :+1: However, before moving on there are some changes that need to be done and I'm listing them below.

- [ ]  There is no aside tag for the editor's pick section. Actually there is no editor's pick section at the rightmost part, at all on your page. Kindly add this.
![aside](https://user-images.githubusercontent.com/33910796/112361844-db9f1480-8ce4-11eb-818c-4be9ca8abaa1.png)

- [ ]  There is no suggested newsletter section above the more in space and astronomy part. Please add this to your page.
- [ ] In the more in space and astronomy part, one picture on the top right is missing. Kindly add this.
- [ ]  As you mentioned in your commit message, your footer is incomplete. They are not aligned on one line and the links at the bottom are not present. Please fix this.
- [ ]  You haven't arranged any readme for your page. I'm giving this [link](https://github.com/microverseinc/readme-template) for help. Kindly add a decent Readme on your repo.

Thanks for your hard work. :muscle: After making those changes, please submit for a re-review.
As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#) you have 2 code reviews left for this project.
If you have any questions about the review, feel free to reach out to me via Slack. :100: 
_Eyüp Sercan UYGUR_

uzairali19 commented:
> ## Status: Changes Required ♻️
> Hello @uzairali19. I've checked your page and it looks good. 👍 However, before moving on there are some changes that need to be done and I'm listing them below.
> 
> * [ ]   There is no aside tag for the editor's pick section. Actually, there is no editor's pick section at the rightmost part, at all on your page. Kindly add this.
>   ![aside](https://user-images.githubusercontent.com/33910796/112361844-db9f1480-8ce4-11eb-818c-4be9ca8abaa1.png)
> * [ ]   There is no suggested newsletter section above the more in space and astronomy part. Please add this to your page.
> * [ ]  In the more in space and astronomy part, one picture on the top right is missing. Kindly add this.
> * [ ]   As you mentioned in your commit message, your footer is incomplete. They are not aligned on one line and the links at the bottom are not present. Please fix this.
> * [ ]   You haven't arranged any readme for your page. I'm giving this [link](https://github.com/microverseinc/readme-template) for help. Kindly add a decent Readme on your repo.
> 
> Thanks for your hard work. 💪 After making those changes, please submit for a re-review.
> As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#) you have 2 code reviews left for this project.
> If you have any questions about the review, feel free to reach out to me via Slack. 💯
> _Eyüp Sercan UYGUR_

Hello Eyüp Sercan UYGUR,

Thank you so much for a detailed review of my project. The reason that you cannot find an aside tag is that I can't see the section on my displays. I have tried on opening the website in different viewports and on different browsers and on my laptop (13inch screen) and my desktop (32inch screen) for I thought it might be hidden for a certain viewport as sometimes the inspector cannot open it. I have tried finding this section for 2 to 3 hours. If you want I can put something else in that you can tell me whatever layout you want me to implement.

I have added the suggested newsletter section in my repo now the only problem was that I accidentally hit "Don't show me this again" and then I couldn't see that, it does open up in incognito. 

> In the more in space and astronomy part, one picture on the top right is missing. Kindly add this.
Can you please elaborate on this? Because if I understand your review correctly, then the "more in space and astronomy" should have a grid of 6 by 6 elements. It currently shows me 2 and 3 total of 5 elements. 2 on the first row and 3 on the second row.

As for the footer it would be done tomorrow. I was trying to deal with these things today. Please clarify these so that i can show you the finished project. Thank you.

eypsrcnuygr commented:
@uzairali19 I believe the original page has 3 by 3 grid, but I'm outside and don't want to make you wait so I'm writing this comment. If the original page has 5 in total, then ignore my comment but as I rememeber it was 6 in total. Please check it carefully and implement as in the original. Lastly, I may review your code or someone else. It depends on the TSE schedule and your submission time. So If I'm gonna be the one, I'd happily review it otherwise good luck.

smcommits reviewed (approved):
# Status: Approved :+1: 
## Hello, @uzairali19! You have done a great job. The website looks remarkable. Nice work addressing the issues pointed out by the previous TSE. This PR is now approved you can go ahead and merge the branch. Congratulations and good luck with your next project. 

## Optional Suggestions
#### Please note that the following suggestions are optional
- [ ] **You should write more descriptive commit messages**
Your commit messages should briefly address the what, why, and how of any changes that you make in a commit. This will make you a great collaborator. Please refer to [this lesson](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) to learn more about making good commits.

uzairali19 commented:
> # Status: Approved 👍
> ## Hello, @uzairali19! You have done a great job. The website looks remarkable. Nice work addressing the issues pointed out by the previous TSE. This PR is now approved you can go ahead and merge the branch. Congratulations and good luck with your next project.
> ## Optional Suggestions
> #### Please note that the following suggestions are optional
> * [ ]  **You should write more descriptive commit messages**
>   Your commit messages should briefly address the what, why, and how of any changes that you make in a commit. This will make you a great collaborator. Please refer to [this lesson](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) to learn more about making good commits.

### Hey,

### Thank you for approving my project. I would make sure to be more careful with carefully committing my messages to make them more descriptive.
